tools/interactive/interactivetool_odv.xml

```diff
@@ -20,8 +20,12 @@
         mkdir -p Documents/ODV/galaxy/data &&
         chmod 777 Documents/ODV/ -R &&
         #if $method.type == 'own_data' :
-            #for $infile in $method.infiles:
-                yes | cp '$infile' './Documents/ODV/galaxy/data/$infile.display_name' &&
+            #for $i, $infile in enumerate($method.infiles):
+                #if $infile.ext == "netcdf":
```

Comment:
@Marie59 something like this I mean ...

Reply:
hmm I am going to test that locally then and I get back here when I succeed ;)

Reply:
It can be that it needs to be adjusted, not tested ... just to explain the idea.

Reply:
element_identifier should probably be modified to remove any whitespace ... in IUC you will find many examples.

Reply:
Oh okay so the last commit you did should be sufficient ?

Reply:
Still not tested, sorry.

Reply:
Okay I am testing

Reply:
Okay for the netcdf par everything works perfectly !! 
However for the tabular part I think it's better to don't force the extension because it can be both txt or csv and I think ODV works differently for this kind of data